django-stubs/db/backends/postgresql/base.pyi

```diff
@@ -15,3 +17,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     _named_cursor_idx: int = ...
     @property
     def pg_version(self) -> str: ...
+
+class CursorDebugWrapper(BaseCursorDebugWrapper):
+    def copy_expert(self, sql: str, file: IOBase, *args: Any): ...
```

Comment:
I see that these methods return something, but it is not clear what 🤔 
https://github.com/django/django/blob/main/django/db/backends/postgresql/base.py#L343-L350

Reply:
Yeah, so this is from psycopg2, not from Django itself - and I could not figure out the type hints of that one.... Partly also because it's a wheel and thus written in C and I'm not familiar with the translation between the c original code and the generated python.... And the documentation doesn't talk about the return type at all...

If you can make sense out of that, that would be amazing^^ I've tried and failed...

psycopg2 documentation: https://www.psycopg.org/docs/cursor.html
implementation of copy_expert: https://github.com/psycopg/psycopg2/blob/master/psycopg/cursor_type.c#L1577